### ISSUE TITLE
Question for the evaluation

### ISSUE THREAD

**Issue by LifeBeyondExpectations** [zswang666/Stereo-LiDAR-CCVNorm#4]
Thank you for sharing the nice work.
I have a question about the evaluation.
In the paper, you train the network with the KITTI_Completition training dataset except for the 142 images that are also included in the KITTI Stereo dataset.

My questions are
1. For the depth evaluation, did you measure the metric using the KITTI Completion validation dataset?

2. For the disparity evaluation, what ground truth did you exploit for the metric computation? The gt from KITTI Stereo Benchmark? or the gt from KITTI Completion Benchmark?

2-1. If you exploit the gt from KITTI Stereo Benchmark for disparity metric computation, how did you employ the raw lidar point-cloud ?? As fas as I know, Some part of the images in KITTI Stereo Benchmark does not have raw point-clouds information.

2-2. If you exploit the gt from KITTI Completion Benchmark for disparity metric computation, did you only use the 142 images that are also included in KITTI Stereo Benchmark??

Thank you.
